data.js

```diff
@@ -4,6 +4,8 @@ Existing tags:
   "Pixel Art"
   "UI"
   "Assets"
+  "Tools"
+  "Team Pick"
```

Comment:
@drewconley @BenjaminRochez How do we feel about "Team Pick"? 
I thought it might be neat to have a little "Team Pick" badge once we have the frontend